express-forms/src/index.js

```diff
@@ -1,17 +1,34 @@
 const express = require('express');
+const bodyParser = require('body-parser');
 
 const app = express();
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
 
-app.get('/', (req, res) => {
-  return res.redirect('/form-with-get');
-});
+app.set('view engine', 'ejs');
+
+app.get('/', (req, res) => res.redirect('/form-with-get'));
+
+app.get('/form-with-get', (req, res) => res.render('form-with-get'));
+
+app.get('/form-with-post', (req, res) => res.render('form-with-post'));
 
-app.get('/form-with-get', (req, res) => {
-  return res.render('form-with-get');
+app.get('/submit-form-with-get', (req, res) => {
+  const { first, last } = req.query;
+  const response = {
```

Comment:
I think we can return `400 Bad Request` if `first` or `last` is not provided in API call.